opm/simulators/linalg/FlowLinearSolverParameters.hpp

```diff
@@ -323,7 +323,7 @@ namespace Opm
             EWOMS_REGISTER_PARAM(TypeTag, std::string, AcceleratorMode, "Use GPU (cusparseSolver or openclSolver) or FPGA (fpgaSolver) as the linear solver, usage: '--accelerator-mode=[none|cusparse|opencl|fpga|amgcl]'");
             EWOMS_REGISTER_PARAM(TypeTag, int, BdaDeviceId, "Choose device ID for cusparseSolver or openclSolver, use 'nvidia-smi' or 'clinfo' to determine valid IDs");
             EWOMS_REGISTER_PARAM(TypeTag, int, OpenclPlatformId, "Choose platform ID for openclSolver, use 'clinfo' to determine valid platform IDs");
-            EWOMS_REGISTER_PARAM(TypeTag, std::string, OpenclIluReorder, "Choose the reordering strategy for ILU for openclSolver and fpgaSolver, usage: '--opencl-ilu-reorder=[level_scheduling|graph_coloring], level_scheduling behaves like Dune and cusparse, graph_coloring is more aggressive and likely to be faster, but is random-based and generally increases the number of linear solves and linear iterations significantly.");
+            EWOMS_REGISTER_PARAM(TypeTag, bool, OpenclIluParallel, "Parallelize ILU decomposition and application on GPU. Default: true");
```

Comment:
Please explain what this means. How is it parallelized?

Reply:
Using level scheduling. When false, it performs serially and is extremely slow, this should only be used for debugging.
We might not actually need runtime access for this setting.

Reply:
I am struggeling a bit.
It is parallel anyway, right just different (more parallel with level scheduling. Should we make that more obvious? E,g,
```
            EWOMS_REGISTER_PARAM(TypeTag, bool, GpuIluLevelScheduling, "Use level scheduling for ILU decomposition and application on GPU for more parallelism. Default: true");
```

Reply:
When false, it is not parallel, every row is done alone.
When true, it is parallelized using level scheduling, which functions the same as before. The difference is that graph coloring is removed.
Grouping GPU params together is fine, but cusparse also uses level scheduling, which cannot be turned off.